cs/src/core/FasterLog/FasterLog.cs

```diff
@@ -19,6 +19,22 @@ namespace FASTER.core
     /// </summary>
     public sealed class FasterLog : IDisposable
     {
+        /// <summary>
+        ///  Represents a entry that can be serialized directly onto FasterLog when enqueuing
+        /// </summary>
+        public interface IEntry
+        {
+            /// <summary></summary>
+            /// <returns> the size in bytes after serialization onto FasterLog</returns>
+            public int SerializedLength();
```

Comment:
should this be GetSerializedLength() or SerializedLength { get; } ?

Reply:
SerializedLength { get; } looks like a good option.